prow-configurations/provision-cluster.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROJECT" ]; then
+      echo "\$PROJECT is empty"
+      exit 1
+fi
+
+if [ -z "$CLUSTER_NAME" ]; then
+      echo "\$CLUSTER_NAME is empty"
+      exit 1
+fi
+
+if [ -z "$ZONE" ]; then
+      echo "\$ZONE is empty"
+      exit 1
+fi
+
+echo "Provisioning cluster '${CLUSTER_NAME}' in project '${PROJECT}' and zone '${ZONE}'"
+
+gcloud container --project "${PROJECT}" clusters create "${CLUSTER_NAME}" \
```

Comment:
IMO number of nodes should be configurable as well with a default value of 2

Reply:
done